### DISCUSSION TITLE
Correlation rule

### DISCUSSION THREAD

**Discussion by CISO-GYM** [wazuh/wazuh#21431] [Q&A]
I want to create a rule that if someone logs in to my firewall and then in 5 mins to my domain controller to notify me and kick him out and block him, how can i do that?

**Comment by 23uk** [comment]
You should use rule with timeframe for this task. 
What rule is triggered when entering your firewall and controller?

**Reply by CISO-GYM** [reply]
5715 and 5501 on each agent
